### ISSUE TITLE
686 | Send marriage termination type due to death to RBPS

### ISSUE THREAD

**Issue by steele-lm** [department-of-veterans-affairs/va.gov-team#92031]
## Issue Description
_A test engineer (Benjamin Brown) with RBPS flagged that one of their tests is failing in the RBPS staging environment bc VA.gov is not transferring the marriage termination type to RBPS when removing a spouse due to death. This causes the EP to sit in a pending state. After some initial investigation, we uncovered that VA.gov does send the marriage termination type for other types of spouse removal (e.g., divorce):_
`marriage_termination_type_code: @divorce_info['reason_marriage_ended'`],

But, VA.gov does not send termination type when removing a spouse due to death:
```
"deaths"=>
    [{"date"=>"2024-01-01",
      "location"=>{"is_outside_us"=>false, "state"=>"ME", "city"=>"portland"},
      "dependent_income"=>false,
      "full_name"=>{"first"=>"test", "last"=>"test"},
      "ssn"=>"987654321",
      "birth_date"=>"1980-01-01",
      "dependent_type"=>"SPOUSE"}],
```

![Image](https://github.com/user-attachments/assets/34cba69f-a863-4b5b-a8e2-4e8f9f465883)

Confirmed with Baha Abu Khaled that RBPS is expecting the value "death" in the marage_trmntn_type_co field.
---
## Acceptance Criteria
- [ ] VA.gov sends RBPS the marriage termination type "Death" for a 686c form when a spouse is removed due to death
- [ ] Benjamin Brown reruns tests and error no longer appears
---

**Comment by steele-lm** [comment]
@evansmith or @mreed101 Any idea if this issue has been resolved with the payload updates and pdf re-mapping that's been done?

**Comment by evansmith** [comment]
I don't think so, I would be curious on the RBPS-end which attribute they expect to be filled when processing a Death for this to reflect properly. We don't consider it a divorce on our end so it wouldnt be sending that type of payload

**Comment by steele-lm** [comment]
RBPS is expecting the value "death".
